client/src/components/Hero.js

```diff
@@ -1,4 +1,7 @@
+/* eslint-disable promise/catch-or-return */
+/* eslint-disable promise/always-return */
```

Comment:
we should probably just fix the rules for these if theyre gonna pop up

Reply:
We're straight up gonna get rid of the example Hero when we start developing our frontend

Reply:
true true  I meant if these are rules you would like to be disabled then do it now

Reply:
all good though non blocker